### ISSUE TITLE
[Feature Request] Only include non-backlog providers in proper-search

### ISSUE THREAD

**Issue by dirkr07** [SiCKRAGETV/sickrage-issues#2391]
Perhaps this is due to a misconfiguration or misunderstanding on my end but this is something I've been noticing lately.

My setup is as follows:
I have 1 main torrent provider that I've set to "enable for daily searches" and 2 others that I've set only for "backlog searches". This way I can be sure that I get my usual quality/screening for my daily searches and should some stuff not be found on that tracker it will jump in to backlog and default back to the other (for me less preffered) trackers.

I've also set them in the order that they should search (first:main - backlog:second/third)

However, even today I've noticed for the latest episode of Suits (S5E06) that it was pulled 3 times from different providers.
First the torrent was snatched from my main (daily search) provider: Ok
Then it seems a proper was released and that was snatched from both my backlog providers.

![image](https://cloud.githubusercontent.com/assets/10185229/8977413/3c91042c-3699-11e5-88a7-00a685b30b90.png)

Is this normal behavior or should I change a setting that I'm missing or a fault in my logic?

If there is something specific I can pull from the logs please let me know.

Thanks for your feedback.

**Comment by adaur** [comment]
You should only get one proper. Can you provide logs for your issue?

**Comment by dirkr07** [comment]
I just had the same/similar issue.
It pulled a proper (only once this time) but from the providers that are set to only search backlog while this imo should've come from the regular daily search provider?

I've included the log from last night/this morning.
It clearly shows that the FIND-PROPERS is going through all my providers instead of just the ones with daily search enabled.
I understand that if a PROPER is released at a later point in time it could be considered a backlog but I would think that at least for the first 24 hrs or so (perhaps a new feature to make this configurable) that it only searches for propers in your providers marked for daily search?

https://gist.github.com/dirkr07/7a2f68d7e108bb3dd768

**Comment by dirkr07** [comment]
Any thoughts?

**Comment by WebSpider** [comment]
I'll look into this later today (dinner first :+1:)

**Comment by WebSpider** [comment]
I've just looked into the code, and from https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/properFinder.py#L77 it's clear that the only criteria for a provider to be included in properSearch is that they have to be active (selected in the UI). If you think this should be different, please send a PR, or a seperate Feature Request if you aren't able to code it yourself.

I don't see any apparent error in the code for detecting multiple PROPERs from the same ep, in https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/properFinder.py#L195 we explicitly check this.

Your current log doesn't show the multiple PROPER issue, please add logfiles showing the odd behaviour

**Comment by dirkr07** [comment]
Well I suggest changing this issue in to a Feature Request (all the info is here) and I'll open a separate issue when I notice I'm pulling multiple propers again

**Comment by OmgImAlexis** [comment]
We have switch to using feathub for feature requests. If this feature is still important to anyone, please
check that it has not been added and add it [HERE](http://feathub.com/SiCKRAGETV/SickRage)
Make sure to vote!
